dev/src/aggregate.ts

```diff
@@ -29,7 +30,7 @@ export class Aggregate {
   constructor(
     readonly alias: AggregateAlias,
     readonly aggregateType: firestore.AggregateType,
-    readonly fieldPath?: string
+    readonly fieldPath?: string | firestore.FieldPath
```

Comment:
FWIW. This argument is only type `FieldPath` in Web. Forcing the API layer to convert string to `FieldPath` earlier. This could be more consistent, but it's not a big deal.

Reply:
That would simplify other code consuming the field path that is calling `FieldPath.fromArgument(this.fieldPath!)`